src/components/ContactSalesForm/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useContext, useState } from 'react'
+import Button from '../../components/common/Button'
+import { ReactComponent as CheckSvg } from '../../img/check.svg'
+import { ReactComponent as CloseSvg } from '../../img/close.svg'
+import { FormState } from '../../types/FormState'
+import { sendEvent } from '../../utils/gtm'
+import classNames from 'classnames'
+import styles from './ContactSalesForm.module.scss'
+import FormContext from '../../context/FormContext'
+import { GATSBY_FPJS_LEAD_URL } from '../../constants/env'
+
+interface ContactSalesFormProps {
+  className?: string | string[]
+}
+export default function ContactSalesForm({ className }: ContactSalesFormProps) {
+  const submitEndpoint = GATSBY_FPJS_LEAD_URL
+
+  const [email, setEmail] = useState('')
+  const [website, setWebsite] = useState('')
+  const { formState, errorMessage, updateFormState, updateErrorMessage } = useContext(FormContext)
+
+  async function handleSubmit(e) {
+    e.preventDefault()
+
+    updateFormState(FormState.loading)
```

Comment:
@borislobanov92 wanted to recommend using a generic hook for forms here. He will write his proposal